static/js/containers/ProfileImage.js

```diff
@@ -80,30 +79,24 @@ class ProfileImage extends React.Component<ProfileImageProps> {
 
   render() {
     const { profile, imageSize, editable, onClick } = this.props
-
-    const imageUrl = makeProfileImageUrl(
-      profile,
-      imageSize === "micro" || imageSize === "small"
-    )
+    let imageUrl
+    if (profile) {
+      imageUrl =
+        imageSize === "micro" || imageSize === "small"
+          ? profile.profile_image_small
+          : profile.profile_image_medium
+    } else {
+      imageUrl = defaultProfileImageUrl
+    }
 
     return (
       <div className="profile-image-container">
         <div className="avatar" onClick={onClick}>
-          {imageUrl.endsWith(defaultProfileImageUrl) &&
-          profile &&
-          profile.name ? (
-              <div className={`profile-image ${imageSize}`}>
-                <div className={`profile-initials ${imageSize}`}>
-                  {initials(profile.name)}
-                </div>
-              </div>
-            ) : (
-              <img
-                src={imageUrl}
-                alt={`Profile image for ${profile ? profile.name : "anonymous"}`}
-                className={`profile-image ${imageSize}`}
-              />
-            )}
+          <img
+            src={imageUrl}
+            alt={`Profile image for ${profile ? profile.name : "anonymous"}`}
```

Comment:
You already have a conditional above that checks for a nil profile object. You can add another variable called `profileName` and determine the value in that `if (profile)` conditional.